.ci-operator.yaml

```diff
@@ -0,0 +1,5 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: rhel-8-release-golang-1.20-openshift-4.14
```

Comment:
is it `1.20` or `1.21` (rhel-8-release-golang-1.21-openshift-4.16) ?

Reply:
This is basically what it is in openshift-release:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/backplane-tools/openshift-backplane-tools-main.yaml#L2-L6

Reply:
As mentioned in the docs:
https://docs.ci.openshift.org/docs/architecture/ci-operator/#build-root-image